mpv/.config/mpv/vs/simulcast.vpy

```diff
@@ -75,7 +72,11 @@ deband_args: Any = dict(iterations=2, threshold=4, radius=16)
 if video_in_dh <= 810:
     deband_args.update(threshold=3, radius=12)
 
+video_in = depth(video_in, 8)
```

Comment:
Wouldn't it make more sense to scale up to 16bit?

Reply:
Done